core/types/header_ext_test.go

```diff
@@ -40,18 +40,23 @@ func TestHeaderRLP(t *testing.T) {
 func TestHeaderJSON(t *testing.T) {
 	t.Parallel()
 
-	testHeaderEncodings(t, json.Marshal, json.Unmarshal)
+	// Note we ignore the returned encoded bytes because we don't
+	// need to compare them to a JSON gold standard.
+	_ = testHeaderEncodeDecode(t, json.Marshal, json.Unmarshal)
 }
 
-func testHeaderEncodings(t *testing.T, encode func(any) ([]byte, error), decode func([]byte, any) error) []byte {
+func testHeaderEncodeDecode(t *testing.T,
+	encode func(any) ([]byte, error),
+	decode func([]byte, any) error) (
+	encoded []byte) {
```

Comment:
```suggestion
func testHeaderEncodeDecode(
	t *testing.T,
	encode func(any) ([]byte, error),
	decode func([]byte, any) error,
) (encoded []byte) {
```

Open and close parentheses, braces, and brackets at the same level of indentation. There is currently no visual cue to the reader w.r.t. a delineation between input args, output args, and implementation.

Reply:
I'm a big fan of one arg per line if all args don't fit in one line.